docs/code/use_dataset_in_paddlepaddle.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
```

Comment:
The last step, you need to squash the commit then use git push -f
1. `git rebase -i HEAD~6`
2. resolve the conflict
3. `git rebase --continue`
you may need to repeat 2,3 operations several times. Notice if the commit message  is correct.
4. `git push -f`

Reply:
Done~
Thank you very much for your meticulous help.